CaPPMS/Pages/Faq.razor

```diff
@@ -1,7 +1,203 @@
-﻿@page "/faq"
+﻿@using System.Collections.Concurrent;
+@using CaPPMS.Data
+@using CaPPMS.Model
+
+@using System.Collections.Generic;
+@using System.IO;
+@using System.Security.Principal;
+@using System.Threading.Tasks;
+
+@using Newtonsoft.Json;
+@using Octokit;
+@using iTextSharp.text;
+@using iTextSharp.text.pdf;
+@using System;
+
+@inject FaqManagerService faqManager
+
+
+@page "/faq"
+<head>
+    <style>
```

Comment:
Style sections in two different spots on the same page? There is another comment about how to do styling below.